apps/condo/domains/organization/integrations/sbbol/SbbolCredentials.js

```diff
@@ -0,0 +1,95 @@
+const querystring = require('querystring')
+const conf = require('@core/config')
+const { getOrganizationAccessToken } = require('./accessToken')
+const { SbbolRequestApi } = require('./SbbolRequestApi')
+const { Organization, TokenSet } = require('@condo/domains/organization/utils/serverSchema')
+const { SBBOL_IMPORT_NAME } = require('./common')
+const dayjs = require('dayjs')
+const { getSchemaCtx } = require('@core/keystone/schema')
+
+const SBBOL_FINTECH_CONFIG = conf.SBBOL_FINTECH_CONFIG ? JSON.parse(conf.SBBOL_FINTECH_CONFIG) : {}
+const SBBOL_PFX = conf.SBBOL_PFX ? JSON.parse(conf.SBBOL_PFX) : {}
+
+class SbbolCredentials {
+
+    context = null
+
+    /**
+     *
+     * @param condoEntryPoint path to entrypoint file relative to current directory
+     * @return {Promise<void>}
+     */
+    async connect () {
+        const { keystone } = await getSchemaCtx('Organization')
+        this.context = await keystone.createContext({ skipAccessControl: true })
+    }
+
+    async getAccessToken () {
+        // `service_organization_hashOrgId` is a `userInfo.HashOrgId` from SBBOL, that used to obtain accessToken
+        // for organization, that will be queried in SBBOL using `SbbolFintechApi`.
+        const result = await getOrganizationAccessToken(SBBOL_FINTECH_CONFIG.service_organization_hashOrgId)
+        return result
+    }
+
+    async changeClientSecret ({ clientId, currentClientSecret, newClientSecret }) {
```

Comment:
it's better to add the `context` as the function argument or as a constructor argument.